utils/validate-user.js

```diff
@@ -52,3 +52,26 @@ exports.validateNewPassword = Joi.object({
     .max(20)
     .required(),
 });
+
+exports.validateSpecialUserRegisterInput = Joi.object({
+  email: Joi.string().email({
+    minDomainSegments: 2,
+  })
+})
+
+exports.validateSpecialUserConfirmInput = Joi.object({
+  firstName: Joi.string()
```

Comment:
Please, reuse rules for the `firstName`, `lastName` and `password` in register / login plain user, register / login admin

Reply:
Fixed